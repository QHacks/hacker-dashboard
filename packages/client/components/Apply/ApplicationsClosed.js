import React from "react";

export default () => (
  <div>
    <p>
      We've closed applications for 2018, but we hope to see you next winter!{" "}
    </p>
    <p>
      <a href="https://qhacks.io">
        Go to our website for information about next year!
      </a>
    </p>
  </div>
);
