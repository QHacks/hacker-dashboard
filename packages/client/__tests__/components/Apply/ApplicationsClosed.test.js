import React from "react";
import ApplicationsClosed from "../../../components/Apply/ApplicationsClosed";
import { shallow } from "enzyme";

const wrapper = shallow(<ApplicationsClosed />);

describe("ApplicationsClosed", () => {
  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
