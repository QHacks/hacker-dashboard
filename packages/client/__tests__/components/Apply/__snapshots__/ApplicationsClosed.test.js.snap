// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationsClosed renders correctly 1`] = `
<div>
  <p
    className="fontSize-large textAlign-center"
  >
    We've closed applications for 2018, but we hope to see you next winter!
     
  </p>
  <p
    className="fontSize-medium textAlign-center"
  >
    <a
      href="https://qhacks.io"
    >
      Back to qhacks.io
    </a>
  </p>
</div>
`;
